moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -200,6 +212,8 @@ class Servo
   std::shared_ptr<const servo::ParamListener> servo_param_listener_;
   planning_scene_monitor::PlanningSceneMonitorPtr planning_scene_monitor_;
 
+  std::optional<KinematicState> last_commanded_state_;
```

Comment:
```suggestion
  KinematicState last_commanded_state_;
```
I think this should not be optional, since `last_commanded_state_ ` should technically always have a value.

Reply:
i'm ok with the optional. Maybe it wouldn't have a value if the program were just started or somebody just called `resetLastCommandedState()`

Reply:
> Maybe it wouldn't have a value if the program were just started or somebody just called `resetLastCommandedState()`

Would it  make sense for `resetLastCommandedState()` to reset it to the current state rather than a `std::nullopt`.
We could then call this same function in the constructor to initialize the `last_commanded_state_`.